tests/publiccloud/check_registercloudguest.pm

```diff
@@ -152,7 +155,8 @@ sub check_instance_unregistered {
         my $creds_output = $instance->ssh_script_output(cmd => 'sudo ls -la /etc/zypp/credentials.d/');
         die("/etc/zypp/credentials.d/ is not empty:\n" . $creds_output);
     }
-    my $out = $instance->ssh_script_output(cmd => 'zypper -t lr ||:', timeout => 300);
+    # my $out = $instance->ssh_script_output(cmd => 'zypper -t lr ||:', timeout => 300);
+    my $out = $instance->zypper_call_remote(cmd => 'lr ||:', global_options => "-t", rc_only => 0, timeout => 300);
```

Comment:
I found that ` 'zypper -t lr ||:'` is more readable than `cmd => 'lr ||:', global_options => "-t"`


```suggestion
    my $out = $instance->zypper_call_remote(cmd => '-t lr ||:', rc_only => 0, timeout => 300);
```

I don't think we need this parameter at all